cmd/pulumi-test-language/tests/policy_config.go

```diff
@@ -67,12 +67,14 @@ func init() {
 					l *L, projectDirectory string, err error, plan *deploy.Plan,
 					changes display.ResourceChanges, events []engine.Event,
 				) {
+					assert.ErrorContains(l, err, "BAIL: step generator errored")
```

Comment:
```suggestion
					require.ErrorContains(l, err, "BAIL: step generator errored")
```

Probably for the others too.  If this is wrong probably the rest of the test doesn't make a lot of sense.